### ISSUE TITLE
Check mod update

### ISSUE THREAD

**Issue by Poil** [FezVrasta/ark-server-tools#804]
Hi,

I use this command to check if I have to update and restart the server
```
/usr/local/bin/arkmanager checkupdate @pve | grep "Your server needs to be restarted"
if [[ $? -eq 0 ]]; then
```
I'm unable to find a good wait to do the same thing for mod update.

Also, checkupdate should return 0 if no update is available

**Comment by klightspeed** [comment]
`checkupdate` uses `EXIT_SUCCESS` (0) if an update is not available, and `EXIT_FAILURE` (1) if an update is available.  You should be able to use:
```
if ! /usr/local/bin/arkmanager checkupdate @pve; then
   ...
fi
```

Unfortunately changing this to use `EXIT_SUCCESS` if an update available would potentially break any scripts relying on the current behaviour.

Commit 586a688 (which will be in 1.6.27) adds a `checkmodupdate` command.

With this, one should be able to use:
```
if /usr/local/bin/arkmanager checkmodupdate @pve; then
    ...
fi
```

**Comment by Poil** [comment]
Thanks !

**Comment by Poil** [comment]
Hey

Return Code seems to be 1 when checking update and none available

```
+ /usr/local/bin/arkmanager checkupdate @pve
Running command 'checkupdate' for instance 'pve'
Querying Steam database for latest version...
Current version:  1914262
Available version:  1914262
Your server is up to date!
+ RET=1
```
